CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.12.0 (unreleased)
```

Comment:
Why add the changelog entry now, in this PR, when an "unreleased" date value makes it seem you don't intend to release immediately?

Reply:
Yeah I jumped the gun on the release changes. Reverted the changelog and version changes. That should come in a separate PR.